### PR TITLE
Better axis labels for examples

### PR DESCRIPTION
**Pull request by oscargus**
## PR Summary

Not so clear which label is which if the same label text is used.

Also, changed the casing of the example name. 1 and 3 are not ideal, but not really sure what a better name should be.

## PR Checklist

**Documentation and Tests**
- [ ] Has pytest style unit tests (and `pytest` passes)
- [ ] Documentation is sphinx and numpydoc compliant (the docs should [build](https://matplotlib.org/devel/documenting_mpl.html#building-the-docs) without error).
- [ ] New plotting related features are documented with examples.

**Release Notes**
- [ ] New features are marked with a `.. versionadded::` directive in the docstring and documented in `doc/users/next_whats_new/`
- [ ] API changes are marked with a `.. versionchanged::` directive in the docstring and documented in `doc/api/next_api_changes/`
- [ ] Release notes conform with instructions in  `next_whats_new/README.rst` or `next_api_changes/README.rst`

### CODE REVIEW

**Review by rcomer — APPROVED** [review]
> 1 and 3 are not ideal, but not really sure what a better name should be.

3 does not seem to modify the axis direction (unless I'm missing something - I'm new to the `axisartist` module).  So maybe "Simple axis direction" and "Simple ticklabel direction"?

**Comment by oscargus** [comment]
Thanks for the suggestion! I applied it with a slight modification and also with a bit of text about the second example.

(I changed to move the ticks to the inside as I think that the default has changed from inside to outside, so no need to move to the outside.)

File: galleries/examples/axisartist/simple_axis_direction03.py [hunk 1]
```diff
@@ -1,8 +1,11 @@
 """
-=======================
-Simple Axis Direction03
-=======================
+==========================================
+Simple axis tick label and tick directions
+==========================================
 
+First subplot moves the tick labels to inside the spines.
+Second subplots moves the ticks to inside the spines.
```
Comment (oscargus) [inline galleries/examples/axisartist/simple_axis_direction03.py:7]: Gah! `s/subplots/subplot/`...

Comment (rcomer) [reply galleries/examples/axisartist/simple_axis_direction03.py:7]: Blame the reviewer!

Comment (QuLogic) [reply galleries/examples/axisartist/simple_axis_direction03.py:7]: Fixed that in #25400.

**Pull request merged**